.github/workflows/tick-ci.yml

```
@@ -0,0 +1,221 @@
+  pull-requests: read
+
+jobs:
+  detect-stage:
+    runs-on: ubuntu-latest
+    outputs:
+      stage: ${{ steps.detect.outputs.stage }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Detect development stage
+        id: detect
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:24: Double quote to prevent globbing and word splitting

(shellcheck)